using-valtimo/document/case-search.md

```diff
@@ -0,0 +1,58 @@
+# Case search
+
+## Search field configuration file
+
+A search field configuration file can be added to the resource folder. These configurations files are automatically
+deployed when the application starts.
+
+Configuration files are only automatically deployed when they are located on this path:
+
+`/resource/config/search/`
+
+The name of the configuration file must match the name of the document definition. This ensures that the search field
+configuration is linked to the correct document definition. For example, if the name of the document definition
+is `profile`, the configuration file name must be:
+
+`/resource/config/search/profile.json`
+
+The content of the configuration file can look like this:
+
+```json
+{
+  "searchFields": [
+    {
+      "key": "customerLastName",
+      "path": "/customer/lastName",
+      "dataType": "text",
+      "fieldType": "single",
+      "matchType": "like"
+    }
+  ]
+}
+```
+
+**key**
```

Comment:
Bit unsure on this formatting. Is this consistent with how we do it for other pages?

Reply:
We don't do this for other pages as far as I know